core/config/blockchain_info.go

```diff
@@ -8,6 +8,24 @@ type BlockchainInfo struct {
 	Blocktime time.Duration
 }
 
+func GetDefaultEtherBasedXChainBlockchains() []uint64 {
+	return []uint64{
+		// Mainnets
+		1,     // Ethereum Mainnet
+		8453,  // Base Mainnet
+		42161, // Arbitrum One
+		10,    // Optimism Mainnet
+
+		// Testnets
+		11155111, // Ethereum Sepolia
+		84532,    // Base Sepolia
+
+		// Local
+		31337, // Anvil Base
+		31338, // Anvil River
+	}
+}
```

Comment:
did you consider adding this to the BlockChainInfo struct and returning it below?

Reply:
Good idea!